docs/page.rst

```diff
@@ -625,6 +626,67 @@ In a nutshell, this is what you can do with PyMuPDF:
 
       PDF only: Insert text into the specified :data:`rect_like` *rect*. See :meth:`Shape.insert_textbox`.
 
+   .. index::
+      pair: rect; insert_htmlbox
+      pair: text; insert_htmlbox
+      pair: css; insert_htmlbox
+      pair: adjust; insert_htmlbox
+      pair: archive; insert_htmlbox
+      pair: overlay; insert_htmlbox
+      pair: rotate; insert_htmlbox
+      pair: oc; insert_htmlbox
+      pair: morph; insert_htmlbox
+
+   .. method:: insert_htmlbox(rect, text, *, css=None, scale_low=0, archive=None, rotate=0, oc=0, overlay=True)
+
+      * New in v1.23.8
+
+      PDF only. Insert text into the specified rectangle. The method has similarities with methods :meth:`Page.insert_textbox` and :meth:`TextWriter.fill_textbox`, but is **much more powerful**. This is achieved by letting a :ref:`Story` object do all the required processing.
+
+      * Parameter "text" may be a string as in the other methods. But it will be **interpreted as HTML source** and may therefore also contain HTML language elements -- including styling. The "css" parameter may be used to pass in additional styling instructions.
```

Comment:
Maybe we can denote a parameter as code like: `text` or `css` instead of like strings?